Bottom: c9161c5501041581316848e3c82971b02f67ed86
Top:    278e7e8d8e54b2110cecdb57882bb072b8902e6e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-05 13:43:31 -0500

Refresh of add-vmdaedalus-inventory

---
diff --git a/Vagrantfile b/Vagrantfile
index a124020d..488140b6 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -92,5 +92,12 @@ Vagrant.configure("2") do |config|
 
         worker.vm.provision "shell", inline: $bbworker
 
+        worker.vm.synced_folder "/mnt/vm/builder",
+            "/mnt/assets",
+            id: "builder-assets",
+            owner: "root",
+            group: "root",
+            mount_options: ["rw"]
+
     end
 end
